pythonScripts/glucoseDataframe.py

```diff
@@ -116,26 +116,108 @@ def createDataframe():
 	date = bolus_carbCsv.loc[:, 'Date']
 	time = bolus_carbCsv.loc[:, 'Time']
 	carb = bolus_carbCsv.loc[:, 'BWZ Carb Input (grams)']
-	header = ['Date', 'Time', 'Bolus (U)', 'Carb Input (grams)']
+	#header = ['Date', 'Time', 'Bolus (U)', 'Carb Input (grams)']
```

Comment:
Just for cleanliness sake if we don't need this line just remove it since our code is so ugly right now haha

Reply:
Oh no need to be sorry. I need to do the same portions of the code I wrote a long time ago haha.